dependencies/config/config_test.go

```diff
@@ -111,9 +111,9 @@ var _ = Describe("Config", func() {
 	})
 
 	It("checks the name of a newly created config file", func() {
-		config, _ := NewConfig(testConfigPath, templatePrefix)
+		Config, _ := NewConfig(testConfigPath, templatePrefix)
```

Comment:
@dkrzyszczyk lower-case "c" is fine here; capital letter indicates exported variable in Golang. Check that error returned from `NewConfig()` is `nil` too.